service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CaseConverterTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, CaseConverter.class})
+class CaseConverterTest {
```

Comment:
I am not sure about these tests, surely it would be better to assert that the expected output is correct rather than just that the object mapper was invoked

Reply:
assertion will be testing object mapper itself (not necessary as @tomaszpowroznik mentioned above) as this method doesn't do much. 

🤷‍♂️🤷‍♂️🤷‍♂️ I personally don't see lot of value for tests in this case